app/redux/ducks/classify.js

```diff
@@ -141,9 +141,18 @@ const TOGGLE_GOLD_STANDARD = 'pfe/classify/TOGGLE_GOLD_STANDARD';
 export default function reducer(state = initialState, action = {}) {
   switch (action.type) {
     case ADD_INTERVENTION: {
+      // this action payload is the json intervention object from sugar
       const intervention = action.payload;
       const { classification } = state;
-      if (classification && classification.links.project === intervention.project_id) {
+      const { project, workflow } = classification.links;
+      const matchesProject = project === String(intervention.project_id);
```

Comment:
`String()` will cast any type to string, so while unlikely, if for some reason whatever we get back from Sugar is not an integer here, do we still want to code to convert that to a string or do we want it to throw? `Number.prototype.toString()` will throw.

Reply:
Also, this will fail if `intervention` is undefined.

Reply:
The deleted line would have failed if `intervention` was undefined too, so I think this is ok.